module/planning/PlanKick/data/config/PlanKick.yaml

```diff
@@ -0,0 +1,18 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+
+# How long since the last ball measurement before ignoring it and assuming the ball is lost, in milliseconds
+timeout_threshold: 1000
+
+# Maximum distance of the ball required to kick the ball (in metres)
+ball_distance_threshold: 0.5
+# Maximum angle of the ball from the torso's x-axis required to kick the ball (in radians), both clockwise and anticlockwise
+ball_angle_threshold: 0.4
+
+# Whether to require aligning with the target
+align: false
```

Comment:
I feel like this is redundant given you have a target angle threshold. Just set that threshold to pi and you have the same effect